### PR TITLE
Update kubelet calico etcd cert paths

### PR DESCRIPTION
**Pull request by tfussell**
Split from https://github.com/giantswarm/k8scloudconfig/pull/801

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/template/master_template.go [hunk 1]
```diff
@@ -300,9 +300,9 @@ systemd:
       Slice=kubereserved.slice
       CPUAccounting=true
       MemoryAccounting=true
-      Environment="ETCD_CA_CERT_FILE=/etc/kubernetes/ssl/etcd/server-ca.pem"
-      Environment="ETCD_CERT_FILE=/etc/kubernetes/ssl/etcd/server-crt.pem"
-      Environment="ETCD_KEY_FILE=/etc/kubernetes/ssl/etcd/server-key.pem"
+      Environment="ETCD_CA_CERT_FILE=/etc/kubernetes/ssl/calico/etcd-ca"
+      Environment="ETCD_CERT_FILE=/etc/kubernetes/ssl/calico/etcd-cert"
+      Environment="ETCD_KEY_FILE=/etc/kubernetes/ssl/calico/etcd-key"
```
Comment (stone-z) [inline pkg/template/master_template.go:305]: I'm confused about the variable naming. For my knowledge, why is the `calico` cert referred to as the `ETCD_X_FILE` / why isn't [the `etcd` bundle](https://github.com/giantswarm/certs/blob/05a37d211a36f3e1bdef367279311f06f6753bf8/pkg/certs/files.go#L44) used?

Comment (tfussell) [reply pkg/template/master_template.go:305]: This is a weird one. On KVM, we use the etcd backend for Calico. This means that each node needs a client certificate to get state directly from etcd. As far as I can tell, this TLS configuration is passed into kubelet with these environment variables. Then the Calico CNI has access to them from the kubelet. I might have this wrong, but I believe Calico fails without them (I tried removing them in the past). This simply aligns the path with our other libraries. The better approach would be to use the kubernetes datastore so we don't have an etcd client cert on every node--don't think we have a story for this yet.

Comment (tfussell) [reply pkg/template/master_template.go:305]: To more directly answer your question, I think that file only exists on the master, but we need it on all of the workers as well so we need to use this one.

**Review by MarcelMue — APPROVED** [review]

**Review by stone-z — APPROVED** [review]

**Comment by tfussell** [comment]
Tested upgrades on KVM and AWS. Going with this.

**Pull request merged**